WizardPanels/WizardPanels.js

```diff
@@ -28,6 +28,11 @@ import css from './WizardPanels.module.less';
 const WizardPanelsContext = createContext(null);
 const DecoratedPanelBase = FloatingLayerIdProvider(PanelBase);
 const HeaderContainer = SpotlightContainerDecorator(Header);
+const ButtonLabel = (props) => (
+	<div className={css.buttonLabel}>
```

Comment:
I don't know the GUI guide, but do we need these 2 div? 3 div are added for each button.

Reply:
The outer one is to make the label inline and the inner one to make the text vertically centered. If there is a better way, please let me know.